### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by junxiant** [warchildmd/ihd#3]
Hello, 

I'm quite interested in your project and i would like to reproduce the results shown in the research paper. Is it possible to share the pre-trained weights so I can evaluate the model on the test set? thank you
